config/locales/views/comments/en.yml

```diff
@@ -28,6 +28,7 @@ en:
         submit: Delete
         edit: Edit
         cancel: Dismiss
+        text_html_comment: Comment deleted
```

Comment:
We tend to only use the `html_` naming when some amount of HTML is going to be passed to the i18n string, which isn't the case for these new strings.

How about we rename this key something like `comment_deleted: Comment deleted`?  Same for the other new key below, perhaps something like `comment_hidden: Comment hidden by post author`?